### PR TITLE
Update REST SortClauses XSD schemas

### PR DESCRIPTION
**Pull request by bdunogier**
Updates the XSD schemas forgotten when SortClauses were implemented in REST (#1687, #1677)

Also changes the definition of sort clause services to match the Symfony standards (lowercase services). Other services will have to be fixed as well, but it will take a bit of effort in order not to break BC.

### CODE REVIEW

**Comment by bdunogier** [comment]
Ping @andrerom. Review appreciated, my XSD ain't very good.

**Comment by andrerom** [comment]
+1

File: doc/specifications/rest/REST-API-V2.rst [hunk 1]
```diff
@@ -7245,17 +7252,16 @@ View XML Schema
...
-            </xsd:complexType>
-          </xsd:element>
-        </xsd:sequence>
+        <xsd:choice minOccurs="1" maxOccurs="unbounded">
+          <xsd:element name="ContentId" type="sortClauseDirectionEnumType" />
+          <xsd:element name="ContentName" type="sortClauseDirectionEnumType" />
+          <xsd:element name="DateModified" type="sortClauseDirectionEnumType" />
+          <xsd:element name="DatePublished" type="sortClauseDirectionEnumType" />
+          <xsd:element name="LocationPath" type="sortClauseDirectionEnumType" />
+          <xsd:element name="LocationPriority" type="sortClauseDirectionEnumType" />
```
Comment (dpobel) [inline doc/specifications/rest/REST-API-V2.rst]: missing LocationDepth, no ?

Comment (bdunogier) [inline doc/specifications/rest/REST-API-V2.rst]: True. Will add.

**Comment by bdunogier** [comment]
Added missing `LocationDepth`, and updated `xsd/Query.xsd` (same changes).

**Comment by dpobel** [comment]
+1

**Comment by bdunogier** [comment]
Failure is unrelated (but real). Merging.

**Pull request merged**